ui/src/ethereum/walletConnect.ts

```diff
@@ -173,7 +173,7 @@ function createConnector(): Connector {
   let modalClosedByWalletConnect = false;
 
   const connector = new Connector({
-    bridge: "https://radicle.bridge.walletconnect.org",
+    bridge: "https://bridge.walletconnect.org",
```

Comment:
I think we can just remove this option.

Reply:
It's still mandatory. If I remove it there's an exception:
```
error.ts?4b5c:174 Error: Missing one of the required parameters: bridge / uri / session
    at new Connector (index.js?4853:32)
    at new WalletConnect (index.js?e07a:5)
    at createConnector (walletConnect.ts?135e:175)
    at WalletConnectClient.reinit (walletConnect.ts?135e:133)
    at new WalletConnectClient (walletConnect.ts?135e:71)
    at createWalletConnect (walletConnect.ts?135e:56)
    at eval (wallet.ts?aa61:106)
    at Module../ui/src/wallet.ts (bundle.js:1)
    at __webpack_require__ (bundle.js:1)
    at eval (transaction.ts:35) {details: {…}}
```

Reply:
The types say it’s optional :facepalm: